starthost-plugin/src/main/java/br/com/autonomiccs/starthost/systemVm/AutonomiccsStartHostSystemVmManager.java

```diff
@@ -357,24 +352,7 @@ public void afterPropertiesSet() throws Exception {
     }
 
     private void copyStartHostApplicationToTempFile() throws FileNotFoundException, IOException {
-        String cloudStackWebappLibFolder = getCloudStackWebappLibFolder();
-        String startHostApplicationFullQualifiedPath = cloudStackWebappLibFolder + "wakeonlan-service-" + wakeOnLanHostApplicationVersion;
-        FileInputStream startHostApplicationStream = new FileInputStream(startHostApplicationFullQualifiedPath);
-        startHostApplicationExecutable = createTempFileForStream(startHostApplicationStream);
-        IOUtils.closeQuietly(startHostApplicationStream);
-    }
-
-    private String getCloudStackWebappLibFolder() {
-        String regexFindCloudStackLibFolder = "jar:file:(.+)\\/.*\\.jar.*";
-        Pattern patternFindCloudStackLibFolder = Pattern.compile(regexFindCloudStackLibFolder);
-
-        String fileUsedToUncoverCloudStackLibFolder = "/application.yml";
-        URL resource = getClass().getResource(fileUsedToUncoverCloudStackLibFolder);
-        Matcher matcher = patternFindCloudStackLibFolder.matcher(ObjectUtils.toString(resource));
-        if(matcher.find()){
-            return matcher.group(1) + "/";
-        }
-        throw new CloudRuntimeException("Could not find cloudstack lib folder.");
+    	startHostApplicationExecutable = AutonomiccsJarAbstraction.WAKEONLAN.getJar();
```

Comment:
now that you are no longer using "wakeOnLanHostApplicationVersion" variable, it should be removed from the "AutonomiccsStartHostSystemVmManager" object. Also, do not foger to remove the "wakeOnLanHostApplicationVersion" object that is created in the applicationContext, the file is "spring-autonomiccsPlugins-context.xml"